cmd/gossamer/flags.go

```diff
@@ -56,9 +56,50 @@ var (
 	// LogFlag cli service settings
 	LogFlag = cli.StringFlag{
 		Name:  "log",
-		Usage: "Supports levels crit (silent) to trce (trace)",
+		Usage: "Global log level. Supports levels crit (silent), eror, warn, info, dbug and trce (trace)",
 		Value: log.LvlInfo.String(),
 	}
+	LogCoreLevelFlag = cli.StringFlag{
+		Name:  "log-core",
+		Usage: "Core package log level. Supports levels crit (silent), eror, warn, info, dbug and trce (trace)",
+		Value: LogFlag.Value,
+	}
+	LogSyncLevelFlag = cli.StringFlag{
+		Name:  "log-sync",
+		Usage: "Sync package log level. Supports levels crit (silent), eror, warn, info, dbug and trce (trace)",
+		Value: LogFlag.Value,
+	}
+	LogNetworkLevelFlag = cli.StringFlag{
+		Name:  "log-network",
+		Usage: "Network package log level. Supports levels crit (silent), eror, warn, info, dbug and trce (trace)",
+		Value: LogFlag.Value,
+	}
+	LogRPCLevelFlag = cli.StringFlag{
+		Name:  "log-rpc",
+		Usage: "RPC package log level. Supports levels crit (silent), eror, warn, info, dbug and trce (trace)",
+		Value: LogFlag.Value,
+	}
+	LogStateLevelFlag = cli.StringFlag{
+		Name:  "log-state",
+		Usage: "State package log level. Supports levels crit (silent), eror, warn, info, dbug and trce (trace)",
+		Value: LogFlag.Value,
+	}
+	LogRuntimeLevelFlag = cli.StringFlag{
+		Name:  "log-runtime",
+		Usage: "Runtime package log level. Supports levels crit (silent), eror, warn, info, dbug and trce (trace)",
+		Value: LogFlag.Value,
+	}
+	LogBlockProducerLevelFlag = cli.StringFlag{
+		Name:  "log-blockproducer",
+		Usage: "Block producer package log level. Supports levels crit (silent), eror, warn, info, dbug and trce (trace)",
+		Value: LogFlag.Value,
+	}
```

Comment:
this is sort of a long name, I'd probably just change it to `log-babe` for now

Reply:
I also renamed `LogBlockProducerLevelFlag` to `LogBabeLevelFlag` for consistency